drivers/CoreLED/tests/CoreLED_test_status.cpp

```diff
@@ -0,0 +1,93 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreLED.h"
+#include "CoreSPI.h"
+#include "gtest/gtest.h"
+
+using namespace leka;
+
+class CoreLedStatusTest : public ::testing::Test
+{
+  protected:
+	CoreLedStatusTest() = default;
+
+	// void SetUp() override {}
+	// void TearDown() override {}
+
+	static constexpr int number_of_leds = 20;
+
+	CoreSPI spi {NC, NC, NC, NC};
+	CoreLED<number_of_leds> leds {spi};
+};
+
+TEST_F(CoreLedStatusTest, isOffAtInstanciation)
+{
+	EXPECT_FALSE(leds.isOn());
+}
+
+TEST_F(CoreLedStatusTest, isOffIfColorSetToBlack)
+{
+	leds.setColor(RGB::black);
+	EXPECT_FALSE(leds.isOn());
+}
+
+TEST_F(CoreLedStatusTest, isOffIfColorIsSetToNotBlack)
+{
+	leds.setColor(RGB::pure_blue);
+	EXPECT_FALSE(leds.isOn());
+}
+
+TEST_F(CoreLedStatusTest, isOffIfShowAtInstanciation)
+{
+	leds.showColor();
+	EXPECT_FALSE(leds.isOn());
+}
+
```

Comment:
Il manque un cas de figure:
```suggestion

TEST_F(CoreLedStatusTest, isOffIfShowAtInstanciationIsSetToNotBlack)
{
	leds.showColor();
	leds.setColor(RGB::pure_blue);
	EXPECT_FALSE(leds.isOn());
}

```

Reply:
Je me suis amusé à relever les différents cas de figure suivant la dernière consigne avant le EXPECT de `leds.isOn()`:

- Hide → `FALSE`
- Show, before set was
    - Missing → `FALSE`
    - Black → `FALSE`
    - Colored (black excluded) → `TRUE`
- Set → Same result as last Show
    - Missing Show → `FALSE`

Reply:
si le cas est déjà là:

```
TEST_F(CoreLedStatusTest, isOffIfColorIsSetToNotBlack)
{
	leds.setColor(RGB::pure_blue);
	EXPECT_FALSE(leds.isOn());
}
```

le showColor sert à rien parce qu'il show la couleur par défault qui est black, ce qui est lui même testé ici 👍 

```
TEST_F(CoreLedStatusTest, isOffIfShowAtInstanciation)
{
	leds.showColor();
	EXPECT_FALSE(leds.isOn());
}
```